flow360/component/geometry.py

```diff
@@ -289,6 +292,44 @@ def show_available_groupings(self, verbose_mode: bool = False):
             show_ids_in_each_group=verbose_mode,
         )
 
+    @classmethod
+    def from_local_storage(cls, mesh_id: str = None, local_storage_path="") -> Geometry:
```

Comment:
```suggestion
    def from_local_storage(cls, geometry_id: str = None, local_storage_path="") -> Geometry:
```

Reply:
Is this function exactly the same as the one in VolumeMeshV2? If so can we consider moving it to the Asset base instead?

Reply:
Let's keep duplicate code to minimum.

Reply:
OK, I move the function to Asset base now.